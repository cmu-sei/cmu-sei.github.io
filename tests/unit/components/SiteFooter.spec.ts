import { shallowMount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import Component from '../../../components/SiteFooter.vue'

describe('SiteFooter.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Component)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
