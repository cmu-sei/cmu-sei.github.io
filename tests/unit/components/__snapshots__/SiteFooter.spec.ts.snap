// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SiteFooter.vue > matches snapshot 1`] = `
"<div data-v-2a8bf75c="" class="bg-gray-900 text-gray-50 py-8">
  <anonymous-stub data-v-2a8bf75c=""></anonymous-stub>
</div>"
`;
